### PR TITLE
Period(".") added.

### PR DESCRIPTION
**Pull request by KrishnanDhanabal**
## Purpose of this pull request

This pull request (PR) 

Custom Grunt configuration files section Period(".") added at the end of file.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/tools/using_grunt.html#using-custom-grunt-configuration-files

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by KrishnanDhanabal** [comment]
@jeff-matthews ok sure.

File: src/guides/v2.2/frontend-dev-guide/translations/xlate.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ For an example of creating a dictionary for a theme for both cases, see [Example
 
 ## Manually translate words and phrases {#m2devgde-xlate-translating}
 
-To translate names, titles, and phrases used in Magento
+To translate names, titles, and phrases used in Magento.
```
Comment (jeff-matthews) [inline src/guides/v2.2/frontend-dev-guide/translations/xlate.md]:
```suggestion
To translate names, titles, and phrases used in Magento:
```

File: src/guides/v2.2/frontend-dev-guide/translations/xlate.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ In addition to the `.csv` file that contains the language dictionary, the langua
 
 *  `composer.json` that contains any dependencies for the language package and a mapping to its defined [locale](https://glossary.magento.com/locale). [Sample composer.json]({{ page.baseurl }}/extension-dev-guide/package/package_module.html#sample-composerjson-file)
```
Comment (jeff-matthews) [inline src/guides/v2.2/frontend-dev-guide/translations/xlate.md]:
```suggestion
*  `composer.json` that contains any dependencies for the language package and a mapping to its defined [locale](https://glossary.magento.com/locale). [Sample composer.json]({{ page.baseurl }}/extension-dev-guide/package/package_module.html#sample-composerjson-file).
```

File: src/guides/v2.3/frontend-dev-guide/translations/xlate.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ For an example of creating a dictionary for a theme for both cases, see [Example
 
 ## Manually translate words and phrases {#m2devgde-xlate-translating}
 
-To translate names, titles, and phrases used in Magento
+To translate names, titles, and phrases used in Magento.
```
Comment (jeff-matthews) [inline src/guides/v2.3/frontend-dev-guide/translations/xlate.md]:
```suggestion
To translate names, titles, and phrases used in Magento:
```

File: src/guides/v2.2/contributor-guide/contributing.md [hunk 4]
```diff
@@ -138,16 +138,16 @@ To verify and test changes completed in a pull request, enter a command to gener
 
 **Admin access:**
 
--  http://pr-xxx.engcom.dev.magento.com/admin
+-  http://pr-xxx.engcom.dev.magento.com/admin.
```
Comment (jeff-matthews) [inline src/guides/v2.2/contributor-guide/contributing.md]:
```suggestion
-  http://pr-xxx.engcom.dev.magento.com/admin
```

Comment (jeff-matthews) [reply src/guides/v2.2/contributor-guide/contributing.md]: A period is not necessary here

File: src/guides/v2.2/contributor-guide/contributing.md [hunk 5]
```diff
@@ -138,16 +138,16 @@ To verify and test changes completed in a pull request, enter a command to gener
...
 -  Admin Credentials:
    -  Username: admin
    -  Password: 123123q
 
 **Permissions:**
 
--  [Community Maintainers](https://github.com/orgs/magento/teams/open-source-maintainers/members)
--  [Magento EngCom Team](https://github.com/orgs/magento/teams/core-maintainers/members)
--  Contributor
+-  [Community Maintainers](https://github.com/orgs/magento/teams/open-source-maintainers/members).
```
Comment (jeff-matthews) [inline src/guides/v2.2/contributor-guide/contributing.md]:
```suggestion
-  [Community Maintainers](https://github.com/orgs/magento/teams/open-source-maintainers/members)
```

Comment (jeff-matthews) [reply src/guides/v2.2/contributor-guide/contributing.md]: A period is not necessary here

File: src/guides/v2.2/contributor-guide/contributing.md [hunk 6]
```diff
@@ -138,16 +138,16 @@ To verify and test changes completed in a pull request, enter a command to gener
...
+-  [Magento EngCom Team](https://github.com/orgs/magento/teams/core-maintainers/members).
```
Comment (jeff-matthews) [inline src/guides/v2.2/contributor-guide/contributing.md]:
```suggestion
-  [Magento EngCom Team](https://github.com/orgs/magento/teams/core-maintainers/members)
```

Comment (jeff-matthews) [reply src/guides/v2.2/contributor-guide/contributing.md]: A period is not necessary here

File: src/guides/v2.2/contributor-guide/contributing.md [hunk 7]
```diff
@@ -138,16 +138,16 @@ To verify and test changes completed in a pull request, enter a command to gener
...
+-  Contributor.
```
Comment (jeff-matthews) [inline src/guides/v2.2/contributor-guide/contributing.md]:
```suggestion
-  Contributor
```

Comment (jeff-matthews) [reply src/guides/v2.2/contributor-guide/contributing.md]: A period is not necessary here

File: src/guides/v2.2/contributor-guide/contributing.md [hunk 8]
```diff
@@ -163,16 +163,16 @@ The command merges the listed related pull requests (`xxx`, `yyy`, `zzz`) into t
...
+-  All mentioned pull requests are open.
+-  All mentioned pull requests have been created by the same contributor (author).
+-  All mentioned pull requests have same target (base) branch.
+-  All mentioned pull requests can be merged with each other.
 
 **Permissions:**
 
--  [Community Maintainers](https://github.com/orgs/magento/teams/open-source-maintainers/members)
--  [Magento EngCom Team](https://github.com/orgs/magento/teams/core-maintainers/members)
+-  [Community Maintainers](https://github.com/orgs/magento/teams/open-source-maintainers/members).
```
Comment (jeff-matthews) [inline src/guides/v2.2/contributor-guide/contributing.md]:
```suggestion
-  [Community Maintainers](https://github.com/orgs/magento/teams/open-source-maintainers/members)
```

Comment (jeff-matthews) [reply src/guides/v2.2/contributor-guide/contributing.md]: A period is not necessary here

File: src/guides/v2.2/contributor-guide/contributing.md [hunk 9]
```diff
@@ -163,16 +163,16 @@ The command merges the listed related pull requests (`xxx`, `yyy`, `zzz`) into t
...
+-  [Magento EngCom Team](https://github.com/orgs/magento/teams/core-maintainers/members).
```
Comment (jeff-matthews) [inline src/guides/v2.2/contributor-guide/contributing.md]:
```suggestion
-  [Magento EngCom Team](https://github.com/orgs/magento/teams/core-maintainers/members)
```

Comment (jeff-matthews) [reply src/guides/v2.2/contributor-guide/contributing.md]: A period is not necessary here

**Pull request merged**